src/main/java/ca/corefacility/bioinformatics/irida/ria/web/projects/ProjectsAjaxController.java

```diff
@@ -74,13 +74,13 @@ public AjaxResponse createNewProject(@RequestBody CreateProjectRequest request,
 	}
 
 	/**
-	 * Get all projects for a user based on a query (searching the name of the project)
+	 * Get a list of projects that the user can share sample to.
 	 *
-	 * @param query To search the project name by
-	 * @return List of project the user has rights to that match the query
+	 * @param currentId Current project identifier
+	 * @return {@link List} of {@link Project}s
 	 */
-	@RequestMapping("/user")
-	public ResponseEntity<List<Project>> getUsersProjects(String query) {
-		return ResponseEntity.ok(projectsService.getProjectsForUser(query));
+	@RequestMapping("/samples-share/projects")
+	public List<Project> getProjectsToShareTo(@RequestParam long currentId) {
```

Comment:
Can we rename this method? To me the name implies that its going to return projects that a sample **is** going to be shared with not projects that a sample **can** be shared with

Reply:
Updated in 5630b02